WatchUI/WatchUI.py

```diff
@@ -1,807 +1,63 @@
-
-    @staticmethod
-    def return_text_from_area(path, page_number: int, x1, y1, x2, y2):
-        """
-        Return text from area. It doesnt use tesseract, so its can be used on normally pdf or without installed
-        tesseract
-        path = Path to pdf
-        page_number = PDF page number where we wanna search for text
-        x1, y1, x2, y2 = coordinates where text is
-        """
-        if os.path.exists(path):
-            doc = fitz.open(path)
-            page = doc[page_number]
-            words_list = page.getTextWords()
-            text = ""
-            xy_numbers = 1
-            for xy in words_list:
-                if float(xy[0]) > float(x1) and float(xy[1]) > float(y1) and float(xy[2]) < float(x2) and \
-                        float(xy[3]) < float(y2):
-                    if xy_numbers == len(words_list):
-                        text += xy[4]
-                    else:
-                        text += xy[4] + " "
-                xy_numbers += 1
-            return text
-        else:
-            raise AssertionError("Can't found file")
-
-    @staticmethod
-    def should_exist_this_text(path, page_number: int, text):
-        """
-        Returns True if <text> is found in page
-        path = Path to pdf
-        page_number = PDF page number where we wanna check text
-        text = Text which we search
-
-        """
-        if os.path.exists(path):
-            doc = fitz.open(path)
-            page = doc[page_number]
-            text_instances = page.searchFor(text)
-            if len(text_instances) > 0:
-                return True
-            else:
-                return False
-        else:
-            raise AssertionError("Can't found file")
+        """Library can be imported either with default output folder and set
+            lowest limit of difference between images (ssim), or
+            you can provide your own values.
+            Keyword Arguments:
+                outputs_folder {str} -- path, where you want to save images with highlighted
+                differences (default: "../Outputs")
+                ssim_basic {float} -- threshold value in the interval (0, 1>. Tests are passed,
+                if ssim value returned by keyword test functions is bigger than this (default: 1.0)
+                format_image {str} -- Format for saving picture/screenshot (png, jpg etc.)
+                tesseract_path {str} -- path for tesseract
+                Example: format_image=jpg (default: png)
+            Examples:
+            | =Setting= | =Value= | =Value= | =Value= | =Comment= |
+            | Library   | WatchUI |      |  | # Uses default values of keyword arguments |
+            | Library   | WatchUI | outputs_folder=<path_to_folder> | | # changes folder to different one |
+            | Library   | WatchUI | outputs_folder=<path_to_folder> | ssim_basic=<float> | # changes output folder and ssim threshold |
+            """
+        super(Image, self).__init__(
```

Comment:
super().__init() should be called inside the subclass __init__() method, or am I mistaken? no wonder, that pylint threw the error, which you supressed ;)

Reply:
Done